### PR TITLE
HarkushaHW3

### PR DESCRIPTION
**Pull request by AndrewHarkusha**

### CODE REVIEW

File: src/com/geekhub/hw3/auth/WrongPasswrodException.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.geekhub.hw3.auth;
+
+/**
+ * Class for WrongPasswrodException
+ */
+public class WrongPasswrodException extends AuthException{
```
Comment (skoryk-oleksandr) [inline src/com/geekhub/hw3/auth/WrongPasswrodException.java:6]: Mistyping in word password

File: src/com/geekhub/hw3/auth/WrongPasswrodException.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  public WrongPasswrodException(){
+      super();
+  }
+
+  public WrongPasswrodException(String exeption){
```
Comment (skoryk-oleksandr) [inline src/com/geekhub/hw3/auth/WrongPasswrodException.java:12]: Mistyping in word Exception

File: src/com/geekhub/hw3/string/StringPerformance.java [hunk 3]
```diff
@@ -12,18 +15,42 @@ public static void main(String[] args) {
 
     private static void testStringPerformance() {
         long beginTime = System.currentTimeMillis();
+        String s1 = "";
+        String s2 = "test";
 
-        //TODO: Do CONCAT_CYCLES times of string concatenation
+
+        for (int i = 0; i<=CONCAT_CYCLES; i++){
+            s1 = s1 + s2;
```
Comment (skoryk-oleksandr) [inline src/com/geekhub/hw3/string/StringPerformance.java:23]: Why don't you want to test also String::concat method

**Comment by skoryk-oleksandr** [comment]
Hometask checked.

File: src/com/geekhub/hw3/auth/WrongPasswrodException.java (hunk 1 above)
Comment (AndrewHarkusha) [inline src/com/geekhub/hw3/auth/WrongPasswrodException.java:6]: Ok. I correct it

File: src/com/geekhub/hw3/string/StringPerformance.java (hunk 3 above)
Comment (AndrewHarkusha) [inline src/com/geekhub/hw3/string/StringPerformance.java:23]: yes, I will change to this method

**Pull request closed without merging**